src/odfesqlodbc/es_parse_result.cpp

```diff
@@ -132,6 +131,14 @@ BOOL CC_No_Metadata_from_ESResult(QResultClass *q_res, ConnectionClass *conn,
                : FALSE;
 }
 
+BOOL CC_Append_Table_Data(json_doc &es_result_doc, QResultClass *q_res,
+                          size_t doc_schema_size, ColumnInfoClass &fields) {
+    ClearError();
+    return AssignTableData(es_result_doc, q_res, doc_schema_size, fields)
```

Comment:
Should this be renamed to Append as well?

Reply:
It is used in other functions with AssignColumnHeaders where this name makes sense. So `CC_Append_Table_Data` works as a wrapper wherever it is used for appending rows.